.github/workflows/test_answers.yml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        exercise: [ex1, ex2, ex3, ex4, ex5, ex6, ex7, ex8, ex9]
+        exercise: [ex1, ex2, ex3, ex4, ex5, ex6, ex7, ex8]
```

```suggestion
      fail-fast: false
```

This will make sure all the answers are always checked. I've just committed this to the main branch as well. GitHub changed the default for this setting a while ago, sorry for not fixing the Gizmo challenge earlier.